iceberg/openhouse/internalcatalog/src/main/java/com/linkedin/openhouse/internal/catalog/OpenHouseInternalCatalog.java

```diff
@@ -34,7 +35,9 @@ public class OpenHouseInternalCatalog extends BaseMetastoreCatalog {
 
   @Autowired HouseTableRepository houseTableRepository;
 
-  @Autowired FileIO fileIO;
+  @Autowired
+  @Qualifier("LegacyFileIO")
```

Comment:
This instance of FileIO is produced by FSStorageProvider path, and we eventually will get rid of it, described in the ["Whats the next plan"](https://github.com/linkedin/openhouse/pull/96). 
We need to qualify all "FileIO" beans otherwise they conflict with each other. I'm open to some other name here..